angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/widget-card.html

```diff
@@ -116,9 +95,9 @@
         </launch-button>
       </div>
 
-      <!-- For simple content portlets, show only an icon -->
+      <!-- For simple content portlets, show only an icon (RARELY USED) -->
```

Comment:
Could "SIMPLE" widgets be refactored to *be* "BASIC" widgets with the `widgetCtrl.renderURL(portlet)` call returning the `/web/widget/{fname}` path?